### DISCUSSION TITLE
Connect a external server

### DISCUSSION THREAD

**Discussion by a2mdigital** [DirectoryTree/LdapRecord#633] [Q&A]
Hello Everyone, my laravel app run at external server, how can i connect with ldap at internal server? I need a create firewall rule to connect a 389 port?

**Comment by stevebauman** [comment]
Hi @a2mdigital,

By external server do you mean outside of its network? If so, then yes you'll have to open up your internal network to outside LDAP requests or; create a VPN to your internal network that your external server connects to.

**Comment by a2mdigital** [comment]
> Hi @a2mdigital,
> 
> By external server do you mean outside of its network? If so, then yes you'll have to open up your internal network to outside LDAP requests or; create a VPN to your internal network that your external server connects to.

Hello Steve, thank you! Yes, it's about outside of network.
